src/resolvers/claims/status-card/statusCard.ts

```diff
@@ -0,0 +1,234 @@
+import { ResolverSubset } from '../../ResolverSubset'
+import {
+  ClaimStatusCard,
+  ClaimStatusCardPill,
+  ClaimStatusCardPillType,
+  ClaimStatusProgressSegment,
+  ClaimStatusProgressType,
+} from '../../../generated/graphql'
+import {
+  ClaimDto,
+  ClaimOutcomeDto,
+  ClaimStatusDto,
+} from '../../../api/upstreams/claimService'
+import { LocalizedStrings } from '../../../translations/LocalizedStrings'
+import { differenceInDays } from 'date-fns'
+
+export const ClaimsStatusCardsSubset: ResolverSubset = {
+  Query: {
+    claims_statusCards: async (
+      _parent,
+      args: { locale: string },
```

Comment:
```suggestion
      { locale },
```
So you don't need to write `args.locale` everywhere 😄